libs/service-portal/health/src/screens/HealthCenterRegistration/HealthCenterRegistration.tsx

```diff
@@ -0,0 +1,279 @@
+              ?.toLowerCase()
+              .includes(filter.toLowerCase()) ||
+            hc.address?.postalCode
+              ?.toLowerCase()
+              .includes(filter.toLowerCase()) ||
+            hc.address?.municipality
+              ?.toLowerCase()
+              .includes(filter.toLowerCase()),
+        )
+
+        if (filteredGroup.length) {
+          acc[key] = filteredGroup
+        }
+
+        return acc
+      },
+      {},
+    )
+  }, [filter, healthCenterGroups])
+
+  if (loading)
+    return (
+      <Box paddingY={2}>
+        <Stack space={2}>
+          <CardLoader />
+        </Stack>
+      </Box>
+    )
+
+  if (error)
+    return (
+      <ErrorScreen
+        tag={formatMessage(m.errorTitle)}
+        title={formatMessage(m.errorFetch)}
+      />
+    )
+
+  return (
+    <Box marginBottom={[6, 6, 10]}>
+      <IntroHeader
+        title={formatMessage(messages.healthCenterRegistrationTitle)}
+        intro={formatMessage(messages.healthCenterRegistrationInfo)}
+      />
+      {errorTransfer && (
+        <Box paddingBottom={4} ref={errorBoxRef}>
+          <AlertMessage
+            type="error"
+            title={formatMessage(
+              messages.healthCenterRegistrationTransferErrorTitle,
+            )}
+            message={formatMessage(
+              messages.healthCenterRegistrationTransferErrorInfo,
+            )}
+          />
+        </Box>
+      )}
+      <Box paddingBottom={4}>
+        <AlertMessage
+          type="warning"
+          message={
+            <Text variant="small">
+              <strong className={styles.strongStyle}>
+                {formatMessage(messages.alert)}
+              </strong>
+              {formatMessage(messages.healthCenterRegistrationWarning)}
+            </Text>
+          }
+        />
+      </Box>
+
+      {!loading && totalCount <= 0 && (
+        <Box width="full" marginTop={4} display="flex" justifyContent="center">
+          <Box marginTop={8}>
+            <EmptyState />
+          </Box>
+        </Box>
+      )}
+
+      <RegisterModal
+        id="healthCenterDialog"
+        title={formatMessage(messages.healthCenterRegistrationModalTitle, {
+          healthCenter: selectedHealthCenter?.name,
+        })}
+        description={formatMessage(messages.healthCenterRegistrationModalInfo)}
+        onClose={() => setSelectedHealthCenter(null)}
+        onAccept={() => handleHealthCenterTransfer()}
+        isVisible={!!selectedHealthCenter}
+        buttonLoading={loadingTransfer}
+      />
+
+      <Box className={styles.filterWrapperStyle} marginBottom={3}>
+        <Input
+          size="sm"
+          onChange={(e) => setFilter(e.target.value)}
+          name="filter"
+          value={filter}
+          placeholder={formatMessage(
+            messages.healthCenterListSearchPlaceholder,
+          )}
+        />
+      </Box>
+
+      {Object.keys(filteredHealthCenters).length ? (
+        <Accordion>
```

Comment:
This should probably be added here, right?

```suggestion
        <Accordion dividerOnTop={false}>
```